packages/phrases/src/locales/zh-cn.ts

```diff
@@ -73,11 +75,29 @@ const translation = {
           description: 'E.g.: Angular, React, Vue',
         },
         traditional: {
-          title: 'Tranditional Web',
+          title: 'Traditional Web',
           subtitle: 'Traditional web app using redirects.',
           description: 'E.g.: Node.js, Express, ASP.NET, Java, PHP',
         },
       },
+      quick_start: {
```

Comment:
will it be better to align wording across components / i18n / folders? i'm seeing `quick_start` and `StartupGuide`

Reply:
Yeah, I just updated component name 10 mins ago, and forgot to update i18n phrases. Good call